eng/dockerfile-templates/sdk/Dockerfile.install-reference-assemblies

```
@@ -0,0 +1,24 @@
+{{
+    set wrap(s) to:{{return cat("'", s, "'")}} ^
+
+    set versions to ["net40", "net45", "net451", "net452", "net46", "net461", "net462", "net47", "net471", "net472", "net48", "net481"] ^
+    set versions to map(versions, wrap) ^
+
+    set packageVersion to VARIABLES["referenceassemblies|version"] ^
+    set packageInstallDir to "packages"
+
+}}# Install Targeting Packs
+RUN powershell -Command "`
+    New-Item -ItemType Directory -Path \"${Env:ProgramFiles(x86)}\Reference Assemblies\Microsoft\Framework\.NETFramework\"; `
+    foreach ($version in @@({{join(versions, ", ")}})) { `
+        # Download reference assembly NuGet package `
+        $package = \"Microsoft.NETFramework.ReferenceAssemblies.${version}\"; `
+        NuGet install \"${package}\" -DirectDownload -ExcludeVersion -Version {{packageVersion}} -OutputDirectory ${Env:TEMP}\Packages; `
```

The casing here is really throwing me off.
```suggestion
        nuget install \"${package}\" -DirectDownload -ExcludeVersion -Version {{packageVersion}} -OutputDirectory ${Env:TEMP}\Packages; `
```